containerize/Dockerfile

```diff
@@ -0,0 +1,49 @@
+ARG BUILDIMAGE
+FROM ${BUILDIMAGE} as builder
+
+ARG BUILD_DATE
+ARG ANALYSIS_NAME=analysis
+LABEL org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.name="$ANALYSIS_NAME Docker image" \
+      org.label-schema.description="Provides a runnable CMSSW image with $ANALYSIS_NAME pre-installed." \
+      org.label-schema.vendor="FNAL"
+
+USER    cmsusr
+WORKDIR /home/cmsusr
+
+ARG CMSSW_VERSION
+ARG TAR
+
+COPY ${TAR} /home/cmsusr/${CMSSW_VERSION}.tar.gz
```

Comment:
Similar to previous comment, could this be:
```Dockerfile
COPY ${TAR} ${WORKDIR}/${CMSSW_VERSION}.tar.gz
```

Reply:
I changed it to `/home/${USER}` as the `${WORKDIR}` and `${HOME}` variables are not available and misleading, respectively.

Reply:
Why is `${WORKDIR}` not available? In fact, it's defined in this file but then never used anywhere...

Reply:
That's not an argument or environment variable. The line:
```
WORKDIR /home/${USER}
```
Is the location the user will be dropped into when they open the container, as opposed to '/'.

Reply:
Ah okay, it's a special Docker directive. In that case, I propose:
```Dockerfile
USERDIR /home/${USER}
WORKDIR ${USERDIR}
```
then `${USERDIR}` can be referenced where useful.

Reply:
Okay, take a look. I think I've implemented what you asked for.

Reply:
Yes, this is better.